src/ontology/foodon-edit.owl

```diff
@@ -16121,7 +16122,7 @@ AnnotationAssertion(dc:date obo:FOODON_00003527 "2020-11-17T23:51:38Z"^^xsd:date
 AnnotationAssertion(rdfs:label obo:FOODON_00003527 "sugarloaf cabbage plant"@en)
 SubClassOf(obo:FOODON_00003527 obo:NCBITaxon_3716)
 
-# Class: obo:FOODON_00003528 (mushroom fruitbody)
+# Class: obo:FOODON_00003528 (mushroom (fruitbody))
```

Comment:
I think you might not have refreshed your github version of foodon?  I took the brackets away from "mushroom (fruitbody)" so if you could make that update that would be good.